portfolio/src/main/webapp/script.js

```diff
@@ -43,4 +43,27 @@ function addRandomQuote() {
     });
 }
 
+/** Adds an HTML element from Servlet */
```

Comment:
nit: Let's add period at end of statement to keep consistent

Reply:
Changes made.